scripts/license.py

```diff
@@ -15,10 +15,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import datetime, sys, re, argparse
-from typing import Dict, Set
+from typing import Dict, Set, List
 
 WIDTH: int = 80
-intel_license: list[str] = [
+intel_license: List[str] = [
```

Comment:
@WangJialei-A Hi, in my local python (3.8.10), `list[str] ` is not allowed. I have updated the script here.